Sources/XMLCoder/Decoder/XMLDecoderImplementation.swift

```diff
@@ -411,10 +419,10 @@ extension XMLDecoderImplementation {
     }
 
     func unbox<T: Decodable>(_ box: Box) throws -> T {
-
+        
```

Comment:
Before we take this in, can we get delete the whitespace changes? 🥶

Reply:
:grimacing: whoops sorry